test/oban/queue/executor_test.exs

```diff
@@ -47,13 +44,6 @@ defmodule Oban.Queue.ExecutorTest do
                |> Executor.resolve_worker()
     end
 
-    test "warning on unexpected return values" do
-      message = capture_log(fn -> %{state: :success} = call_with_mode("warn") end)
-
-      assert message =~ "Expected #{__MODULE__}.Worker.perform/1"
-      assert message =~ "{:bad, :this_will_warn}"
-    end
```

Comment:
I think this doesn't make sense now that we don't log warnings when running on _unsafe_ mode.

Reply:
Agreed 👍